packages/mobile/src/common/enums/product/product-status.ts

```diff
@@ -0,0 +1,6 @@
+enum ProductStatus {
```

Comment:
I believe we should have that enum in `shared`

Reply:
Doesn't exist